sql-native/src/main/java/org/ballerinalang/sql/parameterprocessor/DefaultResultParameterProcessor.java

```diff
@@ -551,7 +551,9 @@ public Object convertTimeStamp(java.util.Date timestamp, int sqlType, Type type)
                     return fromString(timestamp.toString());
                 case TypeTags.OBJECT_TYPE_TAG:
                 case TypeTags.RECORD_TYPE_TAG:
-                    if (type.getName().equalsIgnoreCase(Constants.SqlTypes.DATETIME)
```

Comment:
In this case you can remove the logic check `type.getName().equalsIgnoreCase(Constants.SqlTypes.DATETIME)` as well.

Reply:
I will remove

Reply:
Updated. Commit Id is 1200e15732e9aa11e651ec00907a9b346ece7359